src/traveltime_google_comparison/config.py

```diff
@@ -57,6 +59,13 @@ def parse_args():
         default=DEFAULT_TOMTOM_RPM,
         help="Maximum number of requests sent to TomTom API per minute",
     )
+    parser.add_argument(
```

Comment:
I think it would be good to make providers customisable. So it'll be possible to choose via arguments or config what kind of providers we want to test

Reply:
yeah I've thought of that too, I'll add that in a future PR